### DISCUSSION TITLE
Facing problem to install Mantine notification

### DISCUSSION THREAD

**Discussion by sandeepsre** [mantinedev/mantine#2325] [Feature requests]
Screenshot
![Capture](https://user-images.githubusercontent.com/16818395/188078691-06f12354-2869-4b7d-8248-47fd8705c137.JPG)
 attached

**Comment by zhulien-ivanov** [comment]
All Mantine packages should have the same version.

run `npx npm-check-updates @mantine/* -u` and then `npm install` or `yarn install`.

**Reply by sandeepsre** [reply]
thanks
